webapp/plugins/vue-filters.js

```diff
@@ -82,6 +82,7 @@ export default ({ app = {} }) => {
       return contentExcerpt
     },
     proxyApiUrl: url => {
+      url = url && (url.url || url)
```

Comment:
very unclear to me what's going on here... could we call the input... `input`? and set it as `url` when there is no `input.url`? or am I reading this wrong?